features/internationalization.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Internationalization
+  The CCA DIP access application will have an interface that can be shown in
+  different languages, being English the source language. On this first iteration
+  of the feature, the translation process won't include configuration/tasks to
+  integrate with translation web services like Transifex or Pootle and it will
+  only include English and French languages. This feature only defines the
+  internationalization of the interface, it doesn't add the ability to translate
+  the Collection, Folder and DigitalFile metadata in the database and Elasticseach
+  indexes.
+
+  Scenario: Application user
+    Given an instance running with the internationalization system setup
+    When a user access the interface
+    Then the interface will appear in the user browser default language
+      But it will fallback to English if that language is not available
```

Comment:
Suggested wording: 

But the language will default to English if the user browser's default language is not available